scripts/initializers/account.js

```diff
@@ -0,0 +1,10 @@
+import { initializers } from '@dropins/tools/initializer.js';
```

Comment:
Can we move the initializer functions directly into the corresponding block files?

Reply:
Agree, although reasoning is that there might be multiple blocks where containers from dropin are used, so we are removing the initializer per block mapping here.

Reply:
That's what we had. The problem is that drop-ins referenced in multiple Containers are initialized more than once. Also, some drop-in initializers, such as PDP and Order, will be more complex and have things like langDefs, etc.

Reply:
I think Mark meant something like the following, but as we said on the call this would not map directly since dropins could be in multiple blocks, not just their "named" block.

```
-blocks
  -checkout
    -checkoutBlock.js
    -checkoutInitialize.js
  -cart
    -cartBlock.js
    -cartInitialize.js
```